src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -478,6 +478,128 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify index maintenance (create, refresh) works with globbing patterns.") {
+    withTempPathAsString { testPath =>
+      val absoluteTestPath = PathUtils.makeAbsolute(testPath)
+      val globPath = absoluteTestPath + "/*"
+      val p1 = absoluteTestPath + "/1"
+      import spark.implicits._
+      SampleData.testData
+        .toDF("Date", "RGUID", "Query", "imprs", "clicks")
+        .limit(10)
+        .write
+        .parquet(p1)
+
+      // Create index with globbing pattern.
+      val df =
+        spark.read.option("spark.hyperspace.source.globbingPattern", globPath).parquet(globPath)
+      val indexConfig = IndexConfig("index", Seq("RGUID"), Seq("imprs"))
+      hyperspace.createIndex(df, indexConfig)
+
+      // Check if latest log file contains data source as globbing pattern.
+      var index = latestIndexLogEntry(systemPath, indexConfig.indexName)
+      assert(index.relations.head.rootPaths.head.equals(globPath))
```

Comment:
nit: shall we create `val relation = index.relations.head`?

Reply:
thanks @imback82 ,added